packages/lesswrong/components/hooks/useInitiateConversation.ts

```diff
@@ -16,13 +16,13 @@ export const useInitiateConversation = (props?: { includeModerators?: boolean })
 
   const currentUser = useCurrentUser();
   const { flash } = useMessages();
-  const [userId, setUserId] = useState<string | null>(null);
-  const skip = !currentUser || !userId;
+  const [userIds, setUserIds] = useState<string[] | null>(null);
```

Comment:
Initially I thought these would need to be sorted to avoid accidentally creating duplicates, but the `"userGroupUntitledConversations"` already matches with `$all` rather than matching an exact order so any order is fine